### ISSUE TITLE
Tests fails in BasicSample

### ISSUE THREAD

**Issue by barontxu** [googlesamples/android-testing#128]
I ran BasicSample tests in Meizu and found all tests fails.
The reason is weird that changeText activities set not "Espresso" but "espresso" to textview.
Still wondering why the bug happens.

**Comment by JoseAlcerreca** [comment]
Yeah that's common. It's because your keyboard capitalizes the first letter.

**Comment by barontxu** [comment]
hello Jose, I mean the sample code typed Espresso but what appears in screen is espresso, it may not a problem with capital.

**Comment by piotrek1543** [comment]
@JoseAlcerreca text is already capitalized:

`public static final String STRING_TO_BE_TYPED = "Espresso";`

@barontxu did you checked on other devices or emulators before report issue? It seems to that your phone doesn't capitalize the first letter.

**Comment by jishufeng** [comment]
@barontxu change to other input method app, like google input method, and set the default type is English.
